cmd/bnsd/app/app_test.go

```diff
@@ -161,6 +164,10 @@ func TestApp(t *testing.T) {
 func tagsAsString(pairs []common.KVPair) string {
 	r := make([]string, len(pairs))
 	for i, v := range pairs {
+		if string(v.Key) == "action" {
+			r[i] = "action"
```

Comment:
pp: you are ignoring the `v.Value`. Should be `r[i] = "action:" + string(v.Value)` for a proper impl but the key is good enough for the checks as we have confidence about the value from the decorator test

Reply:
Note that the function for other tags just returns a cleaned up Key, it doesn't attach the Value. But I guess I could do this as well

Reply:
Update: this is only for debug output. Key names is fine enough to check ordering